index.html

```diff
@@ -68,6 +71,64 @@
             </nav>
         </div>
     </div>
-
+    <main class="page-content">
+        <div class="container">
+            <section class="promo-slider">
+                <div class="slider">
+                    <div class="slider__item slider__item--1">
+                        <div class="slider__img-wrap slider__img-wrap--01"></div>
+                        <div class="slider__descr-wrap slider__descr-wrap--1">
```

Comment:
why again -wrap?

Reply:
slider__descr-wrap--1 bad solution, create another element in html and put number to html